.github/workflows/scripts/scanner-update-nvd.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+import os
+import json
+from datetime import datetime, timedelta
+import logging
+from urllib import request, parse
+import time
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(levelname)s: %(message)s")
+
+BASE_URL = "https://services.nvd.nist.gov/rest/json/cves/2.0"
+API_KEY = os.getenv('API_KEY', None)
```

Comment:
Consider failing the script if the API_KEY is not set. 
Consider adding a prefix to the env var name, e.g. `SCANNER_UPDATER_NVD_API_KEY`